Source/abci/Importer/AlembicImporter.h

```diff
@@ -138,6 +141,7 @@ struct aiMeshSummary
     bool has_tangents = false;
     bool has_uv0 = false;
     bool has_uv1 = false;
+    int  has_attributes = 0;
```

Comment:
```suggestion
    int  attributesCount = 0;
```
I know it's going to break the pattern but I prefer to use a more descriptive name.